hw/ip/otbn/doc/_index.md

```diff
@@ -490,6 +490,35 @@ The error that caused the alert can be determined by reading the {{< regref "FAT
 If OTBN was running, this value will also be reflected in the {{< regref "ERR_BITS" >}} register.
 A fatal alert can only be cleared by resetting OTBN through the `rst_ni` line.
 
+
+### Error Prioritization
+
+OTBN can signal multiple errors at once, by setting multiple bits of {{< regref "ERR_BITS" >}}.
+There are some scenarios where one error takes priority over another.
+In these scenarios the higher priority error will suppress some other errors.
+
+* Any error that produces a fatal alert will suppress all non fatal errors.
+* Any `call_stack` error that occurs due to a pop will suppress:
+  - `bad_data_addr`
+  - `bad_insn_addr`
```

Comment:
Oh, I should have said: If I pop from an empty stack when getting e.g. the base address for a `sw`, I assume you have to shoot down the instruction anyway, to avoid a store going out to memory.

Reply:
I guess my point is that we should aim to have an architecture that does "what you'd expect" unless we really can't for some reason. If we pop from an empty stack, does the instruction have any non-error-bits side effects? If not, I think it would be much better to say that, and then give it as the explanation for why the extra error bits aren't set. This is way easier for a reader to reason about, even if it's not a description of how the implementation happens to work.

Reply:
Ok I can rephrase it so it's written that way (e.g. 'a call stack pop occurs before the rest of the instruction executes. If a call stack pop produces a call_stack error no further instruction side effects or errors shall occur, in particular if a 'call_stack' error is seen the following errors will not be seen).